pyi.py

```diff
@@ -362,8 +362,8 @@ def active(self) -> bool:
 
 
 class PyiVisitor(ast.NodeVisitor):
-    def __init__(self, filename: Path = Path("none")) -> None:
-        self.filename = filename
+    def __init__(self, filename: Path | None = None) -> None:
+        self.filename = Path("(none)") if filename is None else filename
```

Comment:
This changes the pathname from `none` to `(none)`. Is this intentional?

Reply:
Yes — it was `(none)` before I (accidentally) changed it in #144; this changes it back, and makes it consistent with `PyiTreeChecker`.

Reply:
(Good catch, though — thanks for checking!)